src/updater/updater.cpp

```diff
@@ -135,12 +156,10 @@ tbb::concurrent_vector<GeometryID>
 updateSegmentData(const UpdaterConfig &config,
                   const extractor::ProfileProperties &profile_properties,
                   const SegmentLookupTable &segment_speed_lookup,
-                  extractor::SegmentDataContainer &segment_data)
+                  extractor::SegmentDataContainer &segment_data,
+                  std::vector<util::Coordinate> coordinates,
```

Comment:
This copies all the coordinates and OSM ids. Should be `const std::vector<util::Coordinate>&` and `const extractor::PackedOSMIDs&`.